ansible/inventory

```diff
@@ -57,6 +57,7 @@ msg.ooni.io
 notify.proteus.ooni.io
 events.proteus.ooni.io
 registry.proteus.ooni.io
+orchestrate.ooni.io
```

Comment:
Sanity: is this host in AMS?

Reply:
Yes. It's actually the same host as events.proteus.ooni.io. After I confirm it deploys cleanly I will remove events.proteus.ooni.io.

Reply:
> Sanity: is this host in AMS?

FYI `inventory-check.yml` has that check.

> It's actually the same host as events.proteus.ooni.io

Please, don't duplicate hosts in inventory.

> After I confirm it deploys cleanly I will remove events.proteus.ooni.io.

I suggest to avoid renames in `inventory` when possible, it leads to mistakes.